src/Schema/Events/Tap.ts

```diff
@@ -92,6 +92,30 @@ export interface TappedArtistSeriesGroup extends TappedEntityGroup {
   destination_screen_owner_type: OwnerType.artistSeries
 }
 
+/**
+ * A user taps a grouping of hero units
+ *
+ *  This schema describes events sent to Segment from [[tappedHeroUnitsGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedHeroUnitsGroup",
+ *    context_module: "home-view-section-hero-units-rail",
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_id: "5359794d1a1e86c3740001f7",
+ *    destination_screen_owner_type: Collection,
+ *    destination_screen_owner_url: "fair/fair-name"
```

Comment:
a hero unit can be a fair, a collection, a show. We mainly get the URL and some text in App to expose. Please let me know if it's fine for me to track the URL here as well to make looking into the data easier.